index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7507e0edea.js" crossorigin="anonymous"></script>
+        <title>Personal portfolio</title>
+</head>
+<body>
+    <div>
+    <nav class="bar">
+       <a href="#"><h2 class="logo">My Logo</h2></a>
+        <div class="icon">
+            <a href="#" alt="bars"><i class="fa-solid fa-bars"></i></a>
+        </div>
+    </nav>
```

Comment:
- [ ] Kindly try to follow the design provided in Figma as much as possible, this includes `font-family`, `font-size`, `color`, `margins` and `paddings`...

## Your design:
![image](https://user-images.githubusercontent.com/50721479/192095453-f623a769-f862-491d-8bf9-3acddf111435.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/192095461-7fcc3462-2f10-4958-880c-de85cda09f08.png)